src/libstd/primitive_docs.rs

```diff
@@ -255,6 +255,11 @@ mod prim_unit { }
 ///
 /// *[See also the `std::ptr` module](ptr/index.html).*
 ///
+/// [`null`]: ../std/ptr/fn.null.html
```

Comment:
To avoid falling back on `core`. I had issues in the past with weird linking like this.